app/services/selection.js

```diff
@@ -97,6 +97,9 @@ export default Service.extend({
   // methods
   handleSummaryLevelToggle(toLevel) {
     const fromLevel = this.get('summaryLevel');
+    if(fromLevel === toLevel) {
+      return;
+    }
```

Comment:
Great catch that this fires on page reload. But ff the `fromLevel` is the same as `toLevel` , why would this cause an explosion? Wouldn't it just explode to the  same level?